thanosql/_base_client.py

```diff
@@ -125,14 +125,30 @@ def _request(
                     message = response_json["error"].get("message", message)
 
                 if code in {400, 405, 422}:
-                    raise thanosql_error.ThanoSQLValueError(message=message)
+                    raise thanosql_error.ThanoSQLValueError(
+                        message=f"Invalid input: {message}"
+                    )
                 elif code in {401, 403}:
-                    raise thanosql_error.ThanoSQLPermissionError(message=message)
+                    raise thanosql_error.ThanoSQLPermissionError(
+                        message=f"Operation not permitted: {message}"
+                    )
                 elif code == 404:
-                    raise thanosql_error.ThanoSQLNotFoundError(message=message)
+                    raise thanosql_error.ThanoSQLNotFoundError(
+                        message=f"Not found: {message}"
+                    )
                 elif code == 409:
-                    raise thanosql_error.ThanoSQLAlreadyExistsError(message=message)
+                    raise thanosql_error.ThanoSQLAlreadyExistsError(
+                        message=f"Object already exists: {message}"
+                    )
+                elif code == 413:
```

Comment:
Caveat: as mentioned in the description, I cannot replicate this error in dev, so I'm not sure how it will work in prod